src/player.cpp

```diff
@@ -4160,14 +4160,14 @@ void player::pause()
     search_surroundings();
 }
 
-void player::shout( std::string msg )
+void player::shout( std::string msg, const std::string &verbal_shout_id )
 {
-    int base = 10;
+    int base = 85;
```

Comment:
Ok, I'll change that back to its default value of 10 (~30 with str muliplier) and leave the multiplier alone. Looking at other sounds, I'd be okay with changing the base to 40-50 perhaps, but 10 seems materially too quiet for a human's shout. I'll just jot down some concerns I have.

The default shout value is about 1.5 to 2 in-game houses in distance. As a young kid, I'd shout to a friend 5 houses down. 2 houses is like a moderately loud yelp. This feature is more of a bellow to call distant survivors.

Local view distance is about 50 tiles during ideal day conditions, and overmap view distance is about 80 tiles. A shout that reaches local view distance seems completely reasonable. I'd even argue a human could shout beyond that in an open, still area like you might expect in a post-apocalyptic setting (save bad weather or loud ambient sounds).

A ~30 shout is generally too quiet to draw zombie hordes since it doesn't extend beyond the local view distance. A shout should draw many nearby hordes.

NPC chatting distance is 24 tiles. Having shout limited to a comparable distance of ~30 makes this feature near useless.

You brought up shriekers shout at 50 volume. That would also seem off for similar reasons above. Also, they can shriek at 120 (50 is the normal attack, it seems). See https://github.com/CleverRaven/Cataclysm-DDA/blob/master/src/monattack.cpp#L285